test_component/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{'name': 'Components Tests',
+ 'summary': 'Automated tests for Components, do not install.',
```

Comment:
Done, Travis already passes the build though. Runbot doesn't for an unknown reason but runbot should not block a merge (the runbot instance is up and running BTW).